README.md

```diff
@@ -80,7 +82,7 @@ This section should be referred to for next steps when any indicator reports a f
 | S0012        | Determines if Puppet produced a report within the last run interval                | https://puppet.com/docs/pe/2021.4/run_puppet_on_nodes.html#troubleshooting_puppet_run_failures                                                                                                                                                                                                                                                                                                                                                                              | Raise a support case quoting reference S0012 along with the output of `puppet agent -td > debug.log 2>&1`                                                                                                                             |
 | S0013        | Determines if a catalog successfully applied on Puppet Agent Last run              | https://puppet.com/docs/pe/2021.4/run_puppet_on_nodes.html#troubleshooting_puppet_run_failures                                                                                                                                                                                                                                                                                                                                                                              | Raise a support case quoting reference S0013 along with the output of  `puppet agent -td > debug.log 2>&1`                                                                                                                            |
-
+| S0030        | Determines if Infrastructure components are running with the setting use_cached_catalog set to true                        | use_cached_catalog should not be configured on PE infrastructure nodes as it prevents the management of key infrastructure settings. Please disable this setting on any infrastructure component https://puppet.com/docs/puppet/7/configuration.html#use-cached-catalog                                                                                                                                                                                                                       | If you are unable, or encounter an error when disabling use_cached_catalog, raise a support case quoting reference S0030 and any errors output when attempting to change the setting                                                                |
```

Comment:
Determines when infrastructure components that run with the setting use_cached_catalog are set to true.

Don't configure use_cached_catalog on PE infrastructure nodes. It prevents the management of key infrastructure settings. 

- Disable this setting on any infrastructure component.
- See: https://puppet.com/docs/puppet/7/configuration.html#use-cached-catalog                                                                                                                                                                                                                       

Submit a support case when you encounter an error, disabling use_cached_catalog.

- Reference S0030.
- Include output errors that appeared when you attempted to change the setting.